src/_gettsim/taxes/zu_verst_eink/freibetraege.py

```diff
@@ -335,27 +335,60 @@ def sonderausgaben_betreuung_y_tu(
     return float(out)
 
 
-def eink_st_kinderfreib_y_tu(
-    anz_kinder_mit_kindergeld_tu: float,
-    anz_erwachsene_tu: int,
+def eink_st_kinderfreib_y(
+    eink_st_kinderfreib_anz_ansprüche: int,
     eink_st_abzuege_params: dict,
 ) -> float:
-    """Aggregate child allowances on tax unit level.
+    """Individual child allowance.
 
     Parameters
     ----------
-    anz_kinder_mit_kindergeld_tu
-        See :func:`anz_kinder_mit_kindergeld_tu`.
-    anz_erwachsene_tu
-        See :func:`anz_erwachsene_tu`.
+    eink_st_kinderfreib_anz_ansprüche
+        See :func:`eink_st_kinderfreib_anz_ansprüche`.
     eink_st_abzuege_params
         See params documentation :ref:`eink_st_abzuege_params <eink_st_abzuege_params>`.
 
     Returns
     -------
 
     """
-    kinderfreib_total = sum(eink_st_abzuege_params["kinderfreib"].values())
-    out = kinderfreib_total * anz_kinder_mit_kindergeld_tu * anz_erwachsene_tu
 
-    return float(out)
+    out = (
+        sum(eink_st_abzuege_params["kinderfreib"].values())
+        * eink_st_kinderfreib_anz_ansprüche
+    )
+    return out
```

Comment:
I don't see much benefit in the creation of a single-use local variable before immediately returning it. But it's a pattern used frequently in the code base. Maybe @hmgaudecker knows why this is being done.